src/bandersnatch/mirror.py

```diff
@@ -79,9 +87,13 @@ def __init__(
         self.json_save = json_save
         self.hash_index = hash_index
         self.root_uri = root_uri
+        self.diff_file = diff_file
+        self.diff_append_epoch = diff_append_epoch
+        self.diff_full_path = diff_full_path
         self.keep_index_versions = keep_index_versions
         self.digest_name = digest_name if digest_name else "sha256"
         self.workers = workers
+        self.diff_file_list = []
```

Comment:
Might want to use your argument. If you set it to none, and it is none, it will be a new empty list with the following suggestion:

```
self.diff_file_list = diff_file_list or []
```